### ISSUE TITLE
Question about the required training epochs

### ISSUE THREAD

**Issue by lymanblue** [ZhaoJ9014/face.evoLVe.PyTorch#37]
Hi~

From the original paper, the required training iterations is 180K for MS1MV2.
By converting this number into training epochs, it would be 512*180K/5.8M = 16 epochs.
(if there are any error in this estimation, please correct me.)

From the provided pre-trained model (IR-50), the training epoch is up to 120.
We would like to know how many training epoch is sufficient for training MS1MV2.

On the other hand, we validate the pre-trained model on megaface (cleaned by deepinsight).
The accuracy is less than 95%. 
Is this result reasonable?

Thank you.

**Comment by ZhaoJ9014** [comment]
Hi, thank you for your question. We released two IR-50 models, one trained with 60 epoches and the other trained with 120 epoches. Our framework is indeed different with ArcFace paper, where you can find the difference of performance on the 7 testing dataset (face.evoLVe performs generally better). As an alternative, we recommend the batch stage configuration like [100000, 140000, 160000] for decreasing the LR using CosFace as head. We are training the model with this configuration and will release soon:)

**Comment by edward0909** [comment]
@lymanblue 
Do you mind sharing your evaluation code on the megaface dataset (thanh1705@hotmail.com)? Thank you!

**Comment by lymanblue** [comment]
Hi~ 
The evaluation code is mainly ported from the insightface repo.
https://github.com/deepinsight/insightface
